doc/whats_new/v0.20.rst

```diff
@@ -216,6 +216,11 @@ Preprocessing
 
 Model evaluation and meta-estimators
 
+- The default of ``n_splits`` parameter of :class:`model_selection.KFold` is
```

Comment:
We have a API changes summary section for it. Also, I think we need a reason for the API change. See e.g.,
```
The default value of ``gamma`` parameter of :class:`svm.SVC`,
:class:`~svm.NuSVC`, :class:`~svm.SVR`, :class:`~svm.NuSVR`,
:class:`~svm.OneClassSVM` will change from ``'auto'`` to ``'scale'`` in
version 0.22 to account better for unscaled features. :issue:`8361` by
:user:`Gaurav Dhingra <gxyd>` and :user:`Ting Neo <neokt>`.
```